pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroupService.java

```diff
@@ -539,29 +549,32 @@ private void initialize() {
                     this.timeUnitScale);
         this.maxIntervalForSuppressingReportsMSecs =
                 this.resourceUsagePublishPeriodInSeconds * this.MaxUsageReportSuppressRounds;
+
     }
 
-    private void checkRGCreateParams(ResourceGroupConfigInfo rgConfig) throws PulsarAdminException {
+    private void checkRGCreateParams(String rgName, org.apache.pulsar.common.policies.data.ResourceGroup rgConfig)
+      throws PulsarAdminException {
         try {
             checkNotNull(rgConfig);
         } catch (NullPointerException e) {
             throw new IllegalArgumentException("ResourceGroupCreate: Invalid null ResourceGroupConfigInfo");
```

Comment:
Nit: `ResourceGroupConfigInfo ` should now be be `ResourceGroup` in the exception string.

Reply:
same, can I take care of it in my next commit.

Reply:
done.